packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,212 @@
+
+![ ](spec-images/combobox-8.png)
+
+Clicking on the menu button opens (if closed) the menu. If the field input contains a value associated with a menu item, selection is indicated on the menu item (not pictured).
+
+![ ](spec-images/combobox-9.png)
+
+Hovering over a menu item in the menu indicates interactivity.
+
+![ ](spec-images/combobox-10.png)
+
+Clicking anywhere on a menu item selects the item, sets the value, displays the value in field input, and closes the menu.
+
+![ ](spec-images/combobox-11.png)
+
+When the menu is open, clicking anywhere outside of a menu item and the text input closes the menu with no changes.
+
+![ ](spec-images/combobox-12.png)
+
+If the combobox allows autocomplete, typing in the field input will show menu items matching the typed value in the field and/or in the menu. The input field autocompletes the first matching menu item. The menu indicates selection on the first matching menu item.
+
+![ ](spec-images/combobox-13.png)
+
+Typing text that does not match any existing menu items either allows the end-user to add a new menu item to the menu or not (see options).
+
+![ ](spec-images/combobox-16.png)
+
+### Keyboard Interactions
+> NOTE: Based on ARIA APG [editable combobox w/ autocomplete](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/)
+
+#### Field Input
+| Key                | Description                                           |
+| ------------------ | ----------------------------------------------------- |
+| `ENTER` | Commits a custom or autocompleted value |
+| `DOWN ARROW` or `UP ARROW` | Opens the menu and moves visual focus to the first or last menu item (DOM focus remains on the combobox) |
+| `ALT` + `DOWN ARROW` | Opens the menu without moving focus or changing selection |
+| `ESC` | Clears the field input value |
+| *Text editing keys* | Behave as expected |
+
+`TAB` focuses the combobox component from the previous or next focusable element on the page.
+
+![ ](spec-images/combobox-17.png)
+
+`DOWN ARROW` or `UP ARROW` on the focused field input opens the menu and focuses (but does not select) the first or last menu item. If the component has inline autocomplete, the value of the focused menu item is populated in the input field. Otherwise, the field input does not change until commit (not pictured).
+
+![ ](spec-images/combobox-18.png)
+
+`ALT` + `DOWN ARROW` on the focused field input opens the menu but does not move visual focus or change selection.
+
+![ ](spec-images/combobox-19.png)
+
+`ENTER` on the focused input field commits the typed, autocompleted, or selected value and closes the menu (if open). The caret remains visible after commit.
+
+![ ](spec-images/combobox-20.png)
+
+`ESC` clears any text in the field input.
+
+![ ](spec-images/combobox-21.png)
+
+Standard single line text editing keys, for example `DELETE` and `ALT` + `RIGHT ARROW`, should work when focus is in the field input (not pictured).
```

Comment:
I don't believe `ALT` + `RIGHT ARROW` is a standard single line text editing key combination. Did you just mean `RIGHT ARROW` (which would just move the cursor one character to the right if the cursor isn't at the end of the text)?

Reply:
Correct, I meant `SHIFT`.